### ISSUE TITLE
Any way to remove inserted css?

### ISSUE THREAD

**Issue by dfcreative** [substack/insert-css#22]
Hi!

I am implementing switchable themes for a package, and use _sheetify_ to compile and insert styles.
I want to change theme dynamically, therefore delete old theme’s css.
Is there any way to do that? Or is it more sheetify’s concern?

**Comment by dfcreative** [comment]
@vvo is there any sense to add `id` param to options, as it is done in [insert-styles](https://github.com/bendrucker/insert-styles)?

**Comment by vvo** [comment]
Currently since insert-css reuse the same `<style>` by default, you could only remove that style element from the DOM, not really what you want maybe?

If you can submit a full proposal for an API change that would allow namespacing multiple insert-css calls and then removing it from the DOM then Ill review it and we can discuss.

Thanks!

**Comment by MickL** [comment]
@vvo I also need the ability to delete a style i added with insert-css and also the ability to update a style i added with insert-css

**Comment by vvo** [comment]
@dy Yes if you submit a proposal API for it and have tests passing etc.. Yep!

**Comment by dy** [comment]
Started working on the PR and there seems to be a no-go.

Now, `insert-css` always appends new css to `<style>` tags corresponding to a container. That perfectly corresponds to the name `insert-`. If we use `id` to allow for reusing `<style>` tags, that will change the behavior from insertion to replacement, which is inconsistent API.

Although it still needs some cosmetic changes.
For example, if we remove newly created `style` tag, `insert-css` does not recognize that removed since it stores that internally:

```js
var style = insertCss('a')
style.parentNode.removeChild(style)
insertCss('b') // reuses old <style> tag, which is not in the DOM anymore
```
